test/functional/app/api/test_user_api.py

```diff
@@ -7,11 +52,14 @@
     (lambda u: u.id, 200),
 ])
 def test_admin_reset_user_pin(
-        test_client, complete_admin_auth_token, create_transfer_account_user, reset_user_id_accessor, status_code
+        test_client, authed_sempo_admin_user, create_transfer_account_user, reset_user_id_accessor, status_code
 ):
+    authed_sempo_admin_user.set_held_role('ADMIN', 'superadmin')
+    auth = get_complete_auth_token(authed_sempo_admin_user)
```

Comment:
need to add this as admin auth is on a module basis, so the changes in `test_edit_user` where affecting tests